src/Microsoft.OData.Edm/Schema/CoreModel/EdmCoreModelComplexType.cs

```diff
@@ -20,68 +20,95 @@ internal sealed class EdmCoreModelComplexType : EdmType, IEdmComplexType, IEdmCo
         /// </summary>
         public static readonly EdmCoreModelComplexType Instance = new EdmCoreModelComplexType();
 
+        /// <summary>
+        /// Gets the base type of this type.
+        /// The Edm.ComplexType is always without base type.
+        /// </summary>
+        public IEdmStructuredType BaseType = null;
+
+        /// <summary>
+        /// private constructor.
+        /// </summary>
+        private EdmCoreModelComplexType()
+        {
+        }
+
         /// <summary>
         /// Gets the kind of this type.
         /// </summary>
-        public override EdmTypeKind TypeKind => EdmTypeKind.Complex;
+        public override EdmTypeKind TypeKind
+        {
+            get { return EdmTypeKind.Complex; }
+        }
 
         /// <summary>
         /// Gets the schema element kind of this type.
         /// </summary>
-        public EdmSchemaElementKind SchemaElementKind => EdmSchemaElementKind.TypeDefinition;

```

Comment:
>=> [](start = 54, length = 2)

something wrong with this syntax?

Reply:
The code was fine, but Stylecop was giving warnings about it and I was not able to suppress the specific warning. So I updated the syntax to get rid of the Stylecop warning.